### DISCUSSION TITLE
How to Add custom dataset class

### DISCUSSION THREAD

**Discussion by nighting0le01** [yoshitomo-matsubara/torchdistill#383] [Q&A]
How to Add custom dataset class such as this:https://github.com/TDeVries/cub2011_dataset/blob/master/cub2011.py
and what modifications would be required to the image_classification.py to get image classfication running on this dataset

Also how to use https://pytorch.org/vision/main/generated/torchvision.datasets.Caltech101.html this dataset inside torchdistill what changes do i need to make to image_classification.py

**Comment by yoshitomo-matsubara** [comment]
At first, make sure that there are no related discussions

See https://github.com/yoshitomo-matsubara/torchdistill/discussions/366#discussioncomment-6392293 https://github.com/yoshitomo-matsubara/torchdistill/discussions/225#discussioncomment-2708307

For Caltech101, you can replace 'CIFAR10' or 'ImageFolder' in sample configs with 'Caltech101' and its kwargs
